config.yaml

```diff
@@ -19,6 +19,17 @@ options:
       Must be one of INFO, WARN, DEBUG, or TRACE, or OFF.
 
   ### Nginx Ingress Integrator-related options:
+  tls-secret-name:
```

Comment:
I am a bit surprised to see `tls-secret-name` as an option to the SDLC charm. I would have expected the only option on the SDLC charm is something like `enable-tls`. As I understand (and as documented here) the only difference that "enabling TLS" with SDLC charm makes is that the callback URLS change to "HTTPS". What am I missing ?

Also is it possible that "enabling TLS" rather than being a config option is determined through relation data. For example if TLS is enabled in the ingress integrator charm then it sets relation data for all related charms saying TLS is enabled. In response any related charm changes its callback URLS. Does this sound reasonable ?

Reply:
Done